### PR TITLE
[fix] ahub issues

### PR DESCRIPTION
**Pull request by lhs8928**
- Added try catch statement
 - Check value to avoid divide by zero
 - Added const keyword to match override function
 - Initialize member variable in constructor

Signed-off-by: hyeonseok lee <hs89.lee@samsung.com>

### CODE REVIEW

**Review by jijoongmoon — APPROVED** [review]
LGTM

File: nntrainer/dataset/databuffer.cpp [hunk 1]
```diff
@@ -55,7 +55,7 @@ class PropsBufferSize : public Property<unsigned int> {
    * @param value default value
    */
   PropsBufferSize(unsigned int value = 1) { set(value); }
-  bool isValid(const unsigned int &v) { return v > 0; }
+  bool isValid(const unsigned int &v) const { return v > 0; }
```
Comment (zhoonit) [inline nntrainer/dataset/databuffer.cpp:58]: could you put override here as well?

**Review by zhoonit — APPROVED** [review]
LGTM :)

File: nntrainer/layers/layer_node.cpp [hunk 2]
```diff
@@ -124,7 +124,8 @@ LayerNode::LayerNode(std::unique_ptr<nntrainer::Layer> &&l) :
   activation_type(ActivationType::ACT_NONE),
   layer_node_props(new PropsType(props::Name(), props::Flatten(),
                                  props::Distribute(), props::Trainable(),
-                                 props::Loss())) {
+                                 props::Loss())),
+  regularization_loss(0.0f) {
```
Comment (kparichay) [inline nntrainer/layers/layer_node.cpp:128]: check indentation here

**Review by kparichay — APPROVED** [review]
LGTM

File: nntrainer/dataset/databuffer.cpp (hunk 1 above)
Comment (lhs8928) [reply nntrainer/dataset/databuffer.cpp]: Done

**Pull request merged**